applications/DEMApplication/python_scripts/DEM_analysis_stage.py

```diff
@@ -210,6 +213,32 @@ def SetRotationalScheme(self):
 
         return rotational_scheme
 
+    def SetRVEUtilities(self):
+        rve_settings = self.DEM_parameters["rve_analysis_settings"]
+        self.rve_analysis                     = rve_settings["RVEAnalysis"].GetBool()
```

Comment:
As I understand, `if not self.rve_analysis`, we do not need to do those operations, right?